src/Compilers/Test/Core/TestBase.cs

```diff
@@ -186,11 +185,6 @@ public virtual void Dispose()
             LazyThreadSafetyMode.PublicationOnly);
         public static MetadataReference MscorlibRef => s_mscorlibRef.Value;
 
-        private static readonly Lazy<MetadataReference> s_mscorlibRefPortable = new Lazy<MetadataReference>(
-            () => AssemblyMetadata.CreateFromImage(ProprietaryTestResources.v4_0_30319.mscorlib_portable).GetReference(display: "mscorlib.v4_0_30319.portable.dll"),
-            LazyThreadSafetyMode.PublicationOnly);
-        public static MetadataReference MscorlibRefPortable => s_mscorlibRefPortable.Value;
```

Comment:
This reference was primarily used for situations in the compiler where we needed a `System.String` type that lacked the `GetEnumerator` method. This setup for `String` is also present in the PortableProfile7 API (though corelib is `System.Runtime.dll` there). Moved compiler tests over to use that.

This was used in the symbol finder tests in the IDE but that doesn't appear to be anything specific to the portable library. Instead it was used because the symbol finder tests need to deal with retargeting between different core libraries. Flipped those to mostly use `net40` and `net461` mscorlib. 
 
https://vstfdevdiv.corp.microsoft.com/DevDiv2/DevDiv/_workitems/edit/667616